packages/common/src/visualizations/types/index.ts

```diff
@@ -47,6 +56,11 @@ export type VizPivotLayoutOptions = {
     reference: string;
 };
 
+export type VizSortBy = {
+    reference: string;
+    type?: SortByType;
```

Comment:
that is to know if we are sorting by `axis` or `bar heights` I was not sure if the reference was pointing to a different axis, in some cases this might be redundant.

Reply:
I'll remove it